platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformExecutionBuildOperationIntegrationTest.groovy

```diff
@@ -217,9 +217,10 @@ class ArtifactTransformExecutionBuildOperationIntegrationTest extends AbstractIn
                 @InputArtifact
                 abstract Provider<FileSystemLocation> getInputArtifact()
 
-                // We need an incremental transform to have a project-bound workspace
-                @Inject
-                abstract InputChanges getInputChanges()
+                ${incremental
```

Comment:
💅 What if we set this in a matrix instead of imperative logic here?

Reply:
Once again I don't feel strongly either way, so I made the change.